src/map/ability.cpp

```diff
@@ -315,7 +315,7 @@ uint16 CAbility::getAoEMsg() const
 
 /************************************************************************
  *                                                                       *
- *  Реализация namespase для работы со способностями                     *
+ *  Namespace implementation for working with abilities                  *
```

Comment:
reminder to future self to wage war against header bricks stating the obvious.

Reply:
I'll also join in this war in further translation commits I make!